cfy_manager/components/nginx/nginx.py

```diff
@@ -273,6 +271,8 @@ def remove(self):
     def start(self):
         logger.notice('Starting NGINX...')
         self._handle_certs()
+        if service._get_service_type() == 'supervisord':
+            service.configure(NGINX, append_prefix=False)
```

Comment:
As above, maybe the logic for this should live in service.configure?

Reply:
Added comments above